core/src/main/resources/lib/form/advanced.jelly

```diff
@@ -44,12 +44,15 @@ THE SOFTWARE.
         <st:adjunct includes="lib.form.advanced.advanced"/>
 
 	    <div class="advancedLink" style="${attrs.align!=null?('text-align:'+attrs.align):''}">
-              <j:set var="id" value="${h.generateId()}"/>
-              <span style="display: none" id="${id}">
-                <l:icon class="icon-notepad icon-md" tooltip="${%customizedFields}"/>
-              </span>
-              <st:nbsp />
-	      <input type="button" value="${attrs.title?:'%Advanced'}..." class="advanced-button advancedButton" /><!-- advancedButton is legacy -->
+            <j:set var="id" value="${h.generateId()}"/>
+
+            <!-- advancedButton is legacy -->
+            <button type="button" class="jenkins-button advanced-button advancedButton">
+                ${attrs.title?:'%Advanced'}...
+                <span id="${id}" tooltip="${%customizedFields}">
```

Comment:
Is there a reason this is at the end of the input whereas others have it at the start?

Reply:
Initial thinking behind putting it on the right was that it's an addition to the button in the case that any fields have been edited, and not visible in every case. Not too sure how I feel about its positioning, I've moved it back to the left side so its consistent with other icons.

Reply:
ahk it's a bit weird anyway

Reply:
Isn't an asterisk a bit too similar to various decorations indicating the creation of something new? E.g. the glare/sun highlights for new document etc. icons in Tango: https://commons.wikimedia.org/wiki/Tango_icons (until I looked it up I was confident it was an actual asterisk!)

Unsure what else to use though. Perhaps something more attention grabbing, like `!` or a similar warning?

Reply:
do people actually know it means something (the symbol inside advanced)? I wasn't aware of what it meant

Reply:
Ah an asterisk is commonly used for when something isn't saved, eg in Notepad, and that was the closest thing I could think of (although our situation is different)

![image](https://user-images.githubusercontent.com/43062514/144608639-30f3a99f-2c67-4466-98bc-bddf7323319c.png)

I'm thinking `!` might be a little too attention grabbing? Wouldn't want the user to think they've done something wrong - it's a hard one

Reply:
what about a wrench or a gear on the left?